docs/resources/User.md

```diff
@@ -132,7 +132,7 @@ The connection object that the user has attached.
 | steam           | Steam               |
 | tiktok          | TikTok              |
 | twitch          | Twitch              |
-| twitter         | Twitter             |
+| twitter         | X                   |
```

Comment:
```suggestion
| twitter         | X (previously known as Twitter |
```

Reply:
I don't think a long description like this is necessary.

```suggestion
| twitter         | Twitter (X)         |
```

Reply:
```suggestion
| twitter         | X (Twitter)         |
```
Wouldn't it make more sense to have Twitter in parenthesis instead of X?

Reply:
I think it's best to match the enum value while also staying accurate to the branding.